site/documentation/content/api/device-connection/index.md

```diff
@@ -126,6 +126,7 @@ The following table provides an overview of the properties a client needs to set
 | :-------------------- | :-------: | :----------------------- | :-------- | :---------- |
 | *subject*             | yes       | *properties*             | *string*  | MUST be set to `set-cmd-handling-adapter-instance`. |
 | *adapter_instance_id* | yes       | *application-properties* | *string*  | The identifier of the protocol adapter instance that currently handles commands for the device or gateway identified by the *device_id* property. |
+| *lifespan*            | yes       | *application-properties* | *int*     | The lifetime of the mapping entry in seconds. After that period, the mapping entry shall be treated as non-existent by the *Device Registration API* methods. A negative value is interpreted as an unlimited lifespan.|
```

Comment:
FMPOV this is an API breaking change. How about making it optional until 2.0.0?

Reply:
Yes, good point. I've changed it accordingly.